Header header

int8 Presence
float32 Reserved1
float32 Reserved2
int32 SignalQuality
